tests/main_test.go

```diff
@@ -260,7 +275,50 @@ func (c *TestContext) CreateNewEncryptedRoom(t *testing.T, creator *client.CSAPI
 				},
 			},
 		},
-	})
+	}
+
+	for _, option := range options {
+		option(&reqBody)
+	}
+
+	return creator.MustCreateRoom(t, reqBody)
+}
+
+type encRoomOptions int
+
+const EncRoomOptions encRoomOptions = 0
```

Comment:
Cute, I like it. This isn't typical Go, which would generally have static functions with prefixes e,g `EncRoomOptionPresetPrivateChat()`.

Reply:
I don't care.